### PR TITLE
Azure Active Directory setup

### PR DESCRIPTION
**Pull request by fdoljanin**
Implementing AAD.
Three types of roles - Printer, Member, Admin; roles per member can be edited in Microsoft Entra by admins.
Still got some things to implement - e.g. route visibility based on role, but that can be solved later.

### CODE REVIEW

File: apps/admin/src/main.tsx [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+import { msalConfig } from './configs/authConfig';
+
+const msalInstance = new PublicClientApplication(msalConfig);
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
   <React.StrictMode>
-    <App />
+    <MsalProvider instance={msalInstance}>
+      <App />
+    </MsalProvider>
```
Comment (bdeak4) [inline apps/admin/src/main.tsx:15]: ovaj provider mozda prebacit u `App.tsx` jer se tamo nalazi i provider za react query pa da su svi na jednom mistu

File: apps/admin/src/configs/authConfig.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export const msalConfig = {
+  auth: {
+    clientId: 'c98e2d15-0979-4dbe-96d1-f8afa75b9207',
```
Comment (bdeak4) [inline apps/admin/src/configs/authConfig.ts:3]: ovo mozda prebacit u .env isto tako da znamo di trazit ako bude bilo potrebe za promjenama

File: apps/admin/src/main.tsx (hunk 1 above)
Comment (fdoljanin) [reply apps/admin/src/main.tsx:15]: image
Ne znam samo gdje stavit ovaj useMsalAuth, on defaultno pozove login ako korisnik nije prijavljen a oslanja se na provider

Comment (bdeak4) [reply apps/admin/src/main.tsx:15]: aha, validno skroz
onda amo prebacit react query provider u main.tsx a da App.tsx bude samo za layout i routing

**Pull request merged**